workshops/models.py

```diff
@@ -108,7 +108,7 @@ class Person(AbstractBaseUser, PermissionsMixin):
     email       = models.CharField(max_length=STR_LONG, unique=True, null=True, blank=True)
     gender      = models.CharField(max_length=1, choices=GENDER_CHOICES, null=True, blank=True)
     may_contact = models.BooleanField(default=True)
-    airport     = models.ForeignKey(Airport, null=True, blank=True)
+    airport     = models.ForeignKey(Airport, null=True, blank=True, on_delete=models.SET_NULL)
```

Comment:
So in my view this should be [`on_delete=models.PROTECT`][1].

[1]: https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.PROTECT